src/bin/cmd/wallet.rs

```diff
@@ -14,7 +14,9 @@
 
 use crate::cmd::wallet_args;
 use crate::config::GlobalWalletConfig;
+/* Turned off for the time being
```

Comment:
should we just remove it?

Reply:
😄   I just copy all those web wallet related switching off.
Ok to remove it, since just one `use`.